livy/src/main/java/org/apache/zeppelin/livy/BaseLivyInterprereter.java

```diff
@@ -69,7 +71,6 @@ public void open() {
     } catch (LivyException e) {
       String msg = "Fail to create session, please check livy interpreter log and " +
           "livy server log";
-      LOGGER.error(msg);
```

Comment:
that likely is because open() calls initLivySession() calls createSession(), so LOGGER is called multiple times in that stack.

how about we keep the LOGGER.error in open()? that would be the last method on the stack that is from this interpreter before returning to the caller.